src/plugins/imagery/components/ImageryView.vue

```diff
@@ -1276,6 +1277,9 @@ export default {
       this.scrollHandler();
     },
     setSizedImageDimensions() {
+      if (!this.$refs.focusedImage) {
```

Comment:
Let's see if we can figure out why this is being called when focusedImage does not exist.

Reply:
The resize observer for `$ref.imageBG` calls `resizeImageContainer()` which in turn calls `setSizedImageDimensions()`.